dashboard/test/lib/school_info_interstitial_helper_test.rb

```diff
@@ -1,173 +1,198 @@
 require 'test_helper'
+require 'timecop'
 
 # Tests following spec:
 # We should pop up this interstitial again every 7 days unless one of the following is true:
 # - The teacher already filled out all the fields (except location)
 # - They chose “Homeschool,” “After School”, "Organization" or “Other” for school type
 # - Teacher chose a country that’s not “United States”
 class SchoolInfoInterstitialHelperTest < ActiveSupport::TestCase
```

Comment:
Note: We rewrote these today to follow the flow diagram posted in the PR description, to make sure we were covering the cases we care about.